OmniKitTests/PodInfoTests.swift

```diff
@@ -133,7 +152,7 @@ class PodInfoTests: XCTestCase {
             XCTAssertEqual(Pod.reservoirLevelAboveThresholdMagicNumber, decoded.reservoirLevel, accuracy: 0.01)
             XCTAssertEqual(8100, decoded.timeActive)
             XCTAssertEqual(TimeInterval(minutes: 0x0087), decoded.timeActive)
-            XCTAssertEqual("02:15", decoded.timeActive.stringValue)
+            XCTAssertEqual("2h15m", decoded.timeActive.timeIntervalStr)
```

Comment:
nit: maybe a little brittle to be asserting debug string format.  I know this was pre-existing; just pointing it out.

Reply:
Agreed on both points, but continuing to keep this long standing code provides a useful reminder that changing the debug string formatting like this has potential implications for other debug and testing programs to consider. If you no longer want to keep this stuff, I’d be willing if needed to delete/replace all these assertions involving the easier to read debug string values for pod time intervals to get this PR landed.